src/app/app.module.ts

```diff
@@ -29,7 +30,8 @@ import {UserService} from './user.service'
     AppComponent,
     LoginComponent,
     SelectRepositoryComponent,
-    CohortBuilderComponent
+    CohortBuilderComponent,
+    SignInComponent
```

Comment:
The tutorial says "In general, the declarations array contains a list of application components, pipes, and directives that belong to the module. A component must be declared in a module before other components can reference it." https://angular.io/tutorial/toh-pt3#declare-herodetailcomponent-in-the-appmodule

https://stackoverflow.com/questions/39062930/what-is-difference-between-declarations-providers-and-import-in-ngmodule says "Selectors of directives, components or pipes are only matched against the HTML if they are declared or imported."

I think the gist here is that, since you want app component to find <sign-in> and attach it to a SignInComponent, you have to tell it about SignInComponent. Maybe the intention is scoping (just importing a component doesn't make it globally visible); slightly annoying, but that model makes sense to me.